workflows/dualrnaseq.nf

```diff
@@ -53,28 +51,18 @@ workflow DUALRNASEQ {
     ch_multiqc_custom_config = params.multiqc_config ? Channel.fromPath(params.multiqc_config) : Channel.empty()
     ch_multiqc_logo = params.multiqc_logo ? Channel.fromPath(params.multiqc_logo) : Channel.empty()
 
-
-
-    // Removed validate samplesheet as had constant issues and errors
-    // If you want to spend the time trying - go ahead
-    ch_reads = ch_samplesheet
```

Comment:
Can we add these comments back in please - this was over half a day mucking around with this!

Reply:
Also - ch_reads = ch_samplesheet - can we add that in - cos in theory, people can pass already trimmed reads into the pipeline